eras/conway/impl/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Version history for `cardano-ledger-conway`
 
+## 1.2.0.0
+
+* Added `ConwayDelegCert` #3372
```

Comment:
Should we also mention `Delegatee`, since it's exposed ?

Reply:
Good catch